doc/changes/DM-53215.feature.rst

```diff
@@ -0,0 +1,3 @@
+When converting a StitchedCoadd to an afw Exposure, it contains a CoaddInputs.
+However, the records do not contain the individual bounding boxes and WCSs.
+Instead, the validPolygon is already in the coadd coordinates.
```

Comment:
This should be mentioned in code comments, too, and any appropriate docs you can find; it's a potential foot-gun, and while I think it's a pragmatic workaround for this problem, we want to mitigate potential harm however we can.